src/Player.ts

```diff
@@ -828,18 +838,21 @@ export class Player {
       } else {
         track.selected = true
         if (track.type === 'audio') {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
I don't see any change related to general linting. Mentioning here even for all other occurrences. Does this maybe fail silently before?

Reply:
Actually I'm gonna remove these ESLint and TS ignore directives and let the errors shine brightly ... the build is successful, I'm not sure right now why the IDE is complaining while the build goes through, but I'll look into that later. This part of the codebase will definitely be revisited later.

As to why it was not necessary before - it was because the typescript typings for PRESTOplay were not complete. Now that I upgraded to the latest version of `castlabs/prestoplay` which is shipped with TS typings, of course a couple of places started to **_light up_**.